lib/src/__tests__/DateTimePicker.test.tsx

```diff
@@ -9,6 +9,23 @@ import { DateTimePicker, DateTimePickerProps } from '../DateTimePicker/DateTimeP
 
 const format = process.env.UTILS === 'moment' ? 'MM/DD/YYYY HH:mm' : 'MM/dd/yyyy hh:mm';
 
+describe('DateTimePicker', () => {
+  describe('prop: mask', () => {
+    it('should handle the mask prop', () => {
+      const component = mount(
+        <DateTimePicker
```

Comment:
I think we need to write bug-pinning test here. Something like "should render custom mask even if `ampm` prop is provided"

```suggestion
        <DateTimePicker 
          ampm
```